UPGRADE.md

```diff
@@ -10,10 +10,25 @@ To upgrade, apply the resource of the latest release. Use the same method that w
 To upgrade to the latest release using `kubectl`, run the following commands:
 
 ```
-$ kubectl apply --server-side -k "https://github.com/piraeusdatastore/piraeus-operator//config/default?ref=v2.6.0"
+$ kubectl apply --server-side -k "https://github.com/piraeusdatastore/piraeus-operator//config/default?ref=v2.7.0"
 $ kubectl wait pod --for=condition=Ready -n piraeus-datastore --all
 ```
 
+# Upgrades from v2.6 to v2.7
+
+Generally, no special steps required.
+
+LINSTOR Satellites now try to automatically detect the hosts LVM configuration. Any patch targeting the Satellites
```

Comment:
`host's LVM configuration` or `LVM configuration on the host`.

Also in Change Log.